exercises/hello-world/HelloWorldSpec.groovy

```diff
@@ -0,0 +1,20 @@
+@Grab('org.spockframework:spock-core:1.0-groovy-2.4')
+import spock.lang.*
+
+class HelloWorldSpec extends Specification {
+
+    def 'outputs "Hello, World!"'() {
+        expect: new HelloWorld().hello() == 'Hello, World!'
```

Comment:
I would have the `expect` and other tags on separate lines.
And I would extract the class instantiation to a field.